homeassistant/components/abode.py

```diff
@@ -34,19 +37,40 @@
     }),
 }, extra=vol.ALLOW_EXTRA)
 
+ABODE_PLATFORMS = [
+    'alarm_control_panel', 'binary_sensor', 'lock', 'switch', 'cover'
+]
+
 
 def setup(hass, config):
     """Set up Abode component."""
+    import abodepy
+
     conf = config[DOMAIN]
     username = conf.get(CONF_USERNAME)
     password = conf.get(CONF_PASSWORD)
 
     try:
-        data = AbodeData(username, password)
-        hass.data[DATA_ABODE] = data
+        abode = abodepy.Abode(username, password)
+        hass.data[DATA_ABODE] = abode
```

Comment:
you can make `hass.data[] = object = bla(init)` that is only a comment not a change request.